README.md

```diff
@@ -11,58 +10,116 @@
+See [different install methods](https://docs.bitsrc.io/en/article/02-install-bit-on-your-computer) for different operation systems.
 
-* Discoverability: it’s hard if not impossible to organize and search multiple repositories and packages to quickly find the components you need. People often used different terms to describe the same functionality, and there is no single source of truth to search and trust.
+### Initialize Bit for your project
 
-### Built for code components
+Initializing Bit on an existing project adds Bit’s virtualization.
 
-Bit solves all of these problems. It adds a level of abstraction on top of your source files, allowing you to create and model components in a virtualized repository. These components can be found and used individually as a dynamic API containing nothing but the code actually used in your application. Bit is designed from the ground to make code components reusable:
+```sh
+bit init
+```
 
-- **Virtual Scope.** Bit uses a distributed and virtual repository called a Scope to keep and maintain all your components in a single place, while still being able to independently find, use and modify each component. You can define the components needed in your application to form a dynamic API made of these components alone, without pulling any redundant code or irrelevant dependencies.
+### Add components
 
-- **Component environment.** Bit lowers the overhead of creating and maintaining multiple reusable components. An isolated configurable environment uses other Bit components (compiler and tester) for transpiling and testing any component using any superset or a testing framework in any context.
+Bit `add` allows to track a subset of files or directories as a reusable code component. Classic use cases would be web components (native, react, angular, etc.), utility functions or any other nodejs module.
 
-- **Component discovery engine.** Bit comes with an integrated search engine that uses expressive linguistic models to make your components discoverable even when you forget the exact name you gave each component. This also helps for collaborating as a team on shared Scopes.
+```sh
+bit add src/utils/left-pad.js
+# Tracked utils/left pad with files 
+```
 
-## Documentation
+You can use glob patterns to track a vast amount of components at once:
 
-[Docs](https://teambit.github.io/bit)
+```sh
+bit add src/utils/*.js
+# Tracked 24 new components
+```
 
-[Bit Scope](https://teambit.github.io/bit/bit-scope.html)
+### Commit
 
-[Bit component](https://teambit.github.io/bit/bit-component.html)
+Bit `commit`, commits changes to new and existing components in your application.
 
-[Bit environment](https://teambit.github.io/bit/bit-component.html#component-environment)
+To check which components were changed or added and about to be committed, you can use `bit status`:
 
-[Bit on the server](https://teambit.github.io/bit/bit-on-the-server.html)
+```sh
+bit status
+# New components:
+#   utils/pad-left
```

Comment:
@itaymendel During the add you used left-pad and here pad-left, it's not consistence.